spec/modules-spec.js

```diff
@@ -47,6 +47,20 @@ describe('third-party module', function () {
       })
     })
   })
+
+  describe('global variables', function () {
+    describe('process', function () {
+      it('can be declared in a module', function () {
+        require(path.join(fixtures, 'module', 'declare-process.js'))
```

Comment:
You could make the path just `'./fixtures/module/declare-process'` if you'd like, would remove the `path.join` calls.